tests/functional/conftest.py

```diff
@@ -7,3 +8,14 @@
 @pytest.fixture
 def test_app(pyramid_settings):
     return webtest.TestApp(create_app(None, **pyramid_settings))
+
+
+@pytest.fixture
+def checkmate_pass(pyramid_settings):
+    httpretty.register_uri(
+        httpretty.GET,
+        "http://localhost:9099/api/check",
+        status=204,
+        body="""{"data": []}""",
```

Comment:
```suggestion
        body=None,
```

Checkmate's 204 responses actually have no body